omnibus.gemspec

```diff
@@ -23,9 +23,9 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency 'chef-sugar',       '~> 3.0'
   gem.add_dependency 'cleanroom',        '~> 1.0'
-  gem.add_dependency 'mixlib-shellout',  '~> 1.4'
+  gem.add_dependency 'mixlib-shellout'
   gem.add_dependency 'mixlib-versioning'
-  gem.add_dependency 'ohai',             '~> 7.2'
+  gem.add_dependency 'ohai'
```

Comment:
This prevents us from properly detecting any platforms added since ohai
7.2, and we have several projects which require new platforms to be
added and supported. Should we be backporting all new platform support
to 7.2 instead?

On 07/23/2015 02:08 PM, Seth Chisamore wrote:
>
> In omnibus.gemspec
> <https://github.com/chef/omnibus/pull/515#discussion_r35359315>:
>
> >    gem.add_dependency 'mixlib-versioning'
> > -  gem.add_dependency 'ohai',             '~> 7.2'
> > +  gem.add_dependency 'ohai'
>
> We should not remove the pessimistic pinning for mixlib-shellout OR ohai.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chef/omnibus/pull/515/files#r35359315>.
>